spec/integration/lib/auth0/api/v2/api_rules_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Rules do
+  attr_reader :client, :enabled_rule, :disabled_rule
+
+  before(:all) do
+    @client = Auth0Client.new(v2_creds)
+    suffix = Faker::Lorem.word
+    script = 'function (user, context, callback) { callback(null, user, context);}'
+    stage = 'login_success'
+    @enabled_rule = client.create_rule("Enabled Rule #{suffix}", script, rand(1..10), true, stage)
+    @disabled_rule = client.create_rule("Disabled Rule #{suffix}", script, rand(11..20), false, stage)
+  end
+
+  after(:all) do
+    rules = client.rules
+    rules.each do |rule|
+      client.delete_rule(rule['id'])
+    end
+  end
+
+  describe '.rules' do
+    let(:rules) { client.rules }
+
+    it { expect(rules.size).to be > 0 }
+
+    context '#filters' do
+      it do
+        expect(client.rules(enabled: true).size).to be 1
+      end
+
+      it do
+        expect(client.rules(enabled: false).size).to be 1
+      end
+
+      it do
+        expect(
+          client.rules(
+            enabled: true,
+            fields: [:script, :order].join(',')).first).to(include('script', 'order'))
```

Comment:
we could move include to a separate line to be consistent with line 46 of same file